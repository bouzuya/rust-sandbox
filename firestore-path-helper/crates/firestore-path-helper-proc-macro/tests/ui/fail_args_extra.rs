fn main() {
    firestore_path_helper_proc_macro::firestore_path_helper!(
        "users/{user_id}",
        user_id = String,
        repo_id = String, // extra argument
    );
}
