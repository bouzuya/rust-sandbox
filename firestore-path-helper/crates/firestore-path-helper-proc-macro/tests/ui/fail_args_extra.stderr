error: argument 'repo_id' does not match any document id segment
 --> tests/ui/fail_args_extra.rs:5:9
  |
5 |         repo_id = String, // extra argument
  |         ^^^^^^^
